### PR TITLE
feat: Add support for mdbook

### PR DESCRIPTION
**Pull request by joekr**
**What this PR does / why we need it**:
This should help simplify downloading mdbook and serving the book
locally.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
No open issue currently related to this PR

### CODE REVIEW

File: docs/build.sh [hunk 1]
```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# Copyright 2019 The Kubernetes Authors.
```
Comment (shyamradhakrishnan) [inline docs/build.sh:3]: please use oracle copyright

Comment (Djelibeybi) [reply docs/build.sh:3]: You cannot change the copyright of existing work. The copyright must remain intact.

Comment (joekr) [reply docs/build.sh:3]: I'll add a comment in the file "build.sh copied unmodified from xxx repository"

**Review by Djelibeybi — APPROVED** [review]

File: docs/build.sh (hunk 1 above)
Comment (Djelibeybi) [reply docs/build.sh:3]: Note that this isn't required, so I've approved this PR as it's fine as-is.

**Pull request merged**